tools/scripts/cc.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#########################################
+# Capture Closed Caption
+#
+# Runs command to listen to the /dev/vbi# device
+# and captures the text to the specified file.
+# Inserts timestamp every 10 seconds.
+#
+# Arguments
+# $1 = output file (default: cc.txt)
+# $2 = device to listen to (default: /dev/vbi0)
+#########################################
+
+output=${$1:-"cc.txt"}
```

Comment:
Run this on the linux box with the Hauppauge TV Capture Card `./cc.sh /path/filename.txt`